src/shared/utils/asset.ts

```diff
@@ -53,21 +44,24 @@ export {
   AssetSynthBCH,
   AssetSynthATOM,
   AssetSynthBSC,
-  AssetBNB,
   AssetBCH,
   AssetATOM,
   AssetLTC,
   AssetDOGE,
   AssetBSC,
   AssetAVAX,
   AssetETH,
-  AssetRune67C,
-  AssetRuneB1A,
   AssetRuneNative,
-  AssetRuneERC20,
-  AssetRuneERC20Testnet,
   AssetSynthEthUsdt,
   AssetKUJI,
   AssetSynthUsk,
   AssetSynthDash
 }
+
+/** Moved from Binance Helpers
+ *
+ */
+export const isMiniToken = ({ symbol }: Pick<Asset, 'symbol'>): boolean => {
```

Comment:
not needed.

Reply:
It is used in other places, e.g. Maya and THORChain to filter assets, so it is needed.

Reply:
This checks to see if the second part od the symbol has a len of 4 and ends with M.
I suspect this is old school BNB specific stuff. Start to remove from  Midgard Services and util.tests